templates/repo/issue/new_form.tmpl

```diff
@@ -236,5 +236,6 @@
 					{{end}}
 				</div>
 		</div>
+		<input type="hidden" name="redirection_after_creation" value="{{.redirection_after_creation}}">
```

Comment:
Why exactly do you provide the redirect link in a hidden input in the UI, although the UI does not use it?
Additionally, the whole redirection seems to be performed on the server-side, so why would you need this unnecessary tag?